### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by isabel22**
Hey guys!

I'm creating a PR with the conclusions of our previous meeting.
I added some text on each of the points, but I would really appreciate if you can review it and if you want to add something else, please feel free to add it in the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
+
+## Responsability
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
+
+## Ownership
+Be the owner of your code and projects.  If something fails on it, you have to take care of it.
+
+## Disagree
```
Comment (Bertiscoding) [inline README.md]: ## Healthy Culture of Controversy
In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+You don't have to agree with everything we say, it is ok to disagree, give your arguments and let's set up an agreement.
+
+## Commit
```
Comment (Bertiscoding) [inline README.md]: ## Disagree and Commit
It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discus it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
+
+## Ownership
+Be the owner of your code and projects.  If something fails on it, you have to take care of it.
+
+## Healthy Culture of Controversy
+In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.
+
+## Disagree and Commit
+Let's commit on what we agree. (please, I need help to finish this one)
```
Comment (isabel22) [inline README.md]:
```suggestion
It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discus it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,29 @@
...
+We all have different backgrounds, beliefs and points of view.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, don't be afraid to be judged. We support each other.
+
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
+
+## Responsability
```
Comment (Bertiscoding) [inline README.md]:
```suggestion
## Responsibility
```

File: README.md [hunk 5]
```diff
@@ -1 +1,29 @@
...
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
+
+## Ownership
+Be the owner of your code and projects.  If something fails on it, you have to take care of it.
```
Comment (Bertiscoding) [inline README.md]:
```suggestion
Be the owner of your code and projects. If something fails on it, you have to take care of it.
```

File: README.md [hunk 6]
```diff
@@ -1 +1,29 @@
...
+We are a team, don't be afraid to be judged. We support each other.
+
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
+
+## Responsibility
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
+
+## Ownership
+Be the owner of your code and projects. If something fails on it, you have to take care of it.
```
Comment (anzak-aleem) [inline README.md]: Does that mean who ever created the bug is responsible to fix it?

File: README.md [hunk 7]
```diff
@@ -1 +1,29 @@
...
+
+## Healthy Culture of Controversy
+In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.
+
+## Disagree and Commit
+It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discuss it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discuss it and make your points during discussion, but once committed you are expected to deliver the highest quality work, specially in broader and shared tasks.
```

File: README.md [hunk 8]
```diff
@@ -1 +1,29 @@
...
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
+
+## Responsibility
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
+
+## Ownership
+Be the owner of your code and projects. If something fails on it, you have to take care of it.
+
+## Healthy Culture of Controversy
+In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Though the person who works on the task has the absolute right to stop the argument at any time.
```

File: README.md [hunk 9]
```diff
@@ -1 +1,29 @@
...
+
+## Respect
+We all have different backgrounds, beliefs and points of view.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, don't be afraid to be judged. We support each other.
+
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
We are the owner of our tasks, and we choose how to do our tasks, when to ask for help, and when to pull the plug on a discussion that we asked for.
```

File: README.md [hunk 10]
```diff
@@ -1 +1,29 @@
...
+
+## Responsibility
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
We are individually responsible for our parts of the products. Also, collectively as a team, we all are responsible for the health and features of whole product.
```

File: README.md (hunk 9 above)
Comment (GomaaK) [reply README.md]: Why we? `We are the owner` doesn't make much sense. What do you want to say here?

File: README.md (hunk 6 above)
Comment (anzak-aleem) [inline README.md]:
```suggestion
Each on of us is the owner of their code and/or projects. If something fails, they have to take care of the failure, ensure completeness and full health of it.
```

File: README.md [hunk 11]
```diff
@@ -1 +1,29 @@
...
+Be the owner of your code and projects. If something fails on it, you have to take care of it.
+
+## Healthy Culture of Controversy
+In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.
+
+## Disagree and Commit
+It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discuss it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.
+
+## Raise the flag
+If you feel that there is something wrong, please raise the flag and speak. Let's have a meeting and talk about the things that you don't like. Your thoughts are important for us.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
If we feel that something is wrong, we should not be afraid to raise the flag and speak about it. 
We have talk about the things that we don't like. Everyone is equally important for us.
```

File: README.md [hunk 12]
```diff
@@ -1 +1,29 @@
...
+
+## Diversity
+We accept others, no matter the gender, nationality, culture, religion.
+
+## Respect
+We all have different backgrounds, beliefs and points of view.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, don't be afraid to be judged. We support each other.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
We are a team, we don't judge rather support each other.
```

File: README.md [hunk 13]
```diff
@@ -1 +1,29 @@
-# team-culture
\ No newline at end of file
+# Code of Conduct
+
+## Diversity
+We accept others, no matter the gender, nationality, culture, religion.
+
+## Respect
+We all have different backgrounds, beliefs and points of view.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
We all have different backgrounds, beliefs, and points of view. We respect all of them.
```

Comment (anzak-aleem) [inline README.md]: This feels like the duplicate of the first one.

Comment (isabel22) [reply README.md]: Maybe, but we should remark both of them. Maybe we should add something else here.

File: README.md (hunk 9 above)
Comment (anzak-aleem) [reply README.md]: I think You tense is just wrong. I know my suggestion also doesn't fix it completely. But something is wrong here. The sentence has to be more elegant. It is rude right now.

File: README.md [hunk 14]
```diff
@@ -1 +1,30 @@
...
+
+## Respect
+We all have different backgrounds, beliefs, and points of view. We respect all of them.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, we don't judge rather support each other.
+
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
```
Comment (Bertiscoding) [inline README.md]:
```suggestion
Each member of the team is the owner of their task. They choose how to do the tasks , when to ask for help, and when to pull the plug on a discussion that they asked for.
```

File: README.md [hunk 15]
```diff
@@ -1 +1,30 @@
...
+
+## Responsibility
+We are individually responsible for our parts of the products. Also, collectively as a team, we all are responsible for the health and features of the whole product.
+
+## Ownership
+Each one of us is the owner of their code and/or projects. If something fails, they have to take care of the failure, ensure completeness and ensure the return to full health.
+
+## Healthy Culture of Controversy
+It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's views might be equally valid and valuable. Although the person who works on the task gets to have the absolute right to stop the argument at any time.
```
Comment (Bertiscoding) [inline README.md]:
```suggestion
It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's views are equally valid and valuable. Although the person who works on the task gets to have the absolute right to stop the argument at any time.
```

File: README.md [hunk 16]
```diff
@@ -1 +1,30 @@
...
+
+## Respect
+We all have different backgrounds, beliefs, and points of view. We respect all of them.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, we don't judge rather support each other.
+
+## Autonomy
+Each member of the team is the owner of their task. They choose how to do the tasks , when to ask for help, and when to pull the plug on a discussion that they asked for.
```
Comment (anzak-aleem) [inline README.md]:
```suggestion
Each member of the team is the owner of their tasks. They choose how to do their tasks, when to ask for help, and when to pull the plug on a discussion that they asked for.
```

**Pull request merged**